lib/claim_letters/claim_letter_downloader.rb

```diff
@@ -7,7 +7,13 @@ class ClaimLetterDownloader
     FILENAME = 'ClaimLetter'
     # 184: Notification Letter (e.g. VA 20-8993, VA 21-0290, PCGL)
     # 339: Rating Decision Letter
-    DOC_TYPE_ALLOWLIST = %w[184].freeze
+    # 27: Board Of Appeals Decision Letter
+    DOC_TYPE_ALLOWLIST =
+      if Flipper.enabled?(:cst_include_ddl_boa_letters, @current_user)
```

Comment:
I was thinking about this - do we even need a feature flag here? This seems like an incremental update to the existing DDL feature that already exists. It adds complexity and the technical risk is minimal.

Tagging @jacobworrell @jerekshoe @samcoforma for input. Thoughts?

Reply:
@kjduensing per our convo, could I get an approval on this?

Reply:
I think we answered this indirectly through yesterday's demo: we're not entirely sure what the PDF version of the letter contains, so we want to hide this behind a feature flag to verify